### PR TITLE
#Task10 : Fix/fix-comment

### PR DESCRIPTION
**Pull request by TheHienDuong**
#Task10 : Update code 
Comment:
image
image
image
image
image
image
image
image

### Overview
Fix:
- Added effects for button
- Add hover and active for a tags
- More transition : before for the tag a
image
- Add active for button
image
-Add effects for service tag
image
-Add select for button 
image

### CODE REVIEW

File: practice-01/index.html [hunk 1]
```diff
@@ -115,7 +115,7 @@ <h5 class="card__title">Sales</h5>
                             class="product__pic">
                     </div>
                     <h3 class="card__text">The best cities for sales careers in 2022</h3>
-                    <h4 class="card__footer">20 Mar 2022, <span class="card__footer--gray"> Bill Holloway</span></h4>
+                    <h4 class="card__footer">20 Mar 2022, <span class="card__footer--gray">by Bill Holloway</span></h4>
```
Comment (ThanhPhat1080) [inline practice-01/index.html]: You can create a css class utility `text--gray` then use it in any text that have color `--west_chester-grey`. Btw, `grey` or `gray` plz pick only one.

Comment (ThanhPhat1080) [reply practice-01/index.html]: Don't foget update all similar cases. Ex: `text--white` , `text--dark`, .....

File: practice-01/index.html [hunk 2]
```diff
@@ -340,57 +340,56 @@ <h4 class="card__footer">
                         and marketing solution for SMBs</p>
                  <ul class="media menu__list">
                         <li>
-                            <img src="./img/Twitter.png" alt="Here is a logo of the media above" class="media-icon">
+                            <a class="item" href="_blank"><img src="./img/Twitter.png" alt="Here is a logo of the media above" class="media-icon"></a>
```
Comment (ThanhPhat1080) [inline practice-01/index.html]: Please navigate to twitter, facebook, linkedid as well. use their homepage

File: practice-01/styles/font.css [hunk 3]
```diff
@@ -16,13 +16,26 @@
   line-height: var(--line-height-xxx-large);
 }
 
+/* text */
+.text--grey{
```
Comment (phattruong-agilityio) [inline practice-01/styles/font.css]: code style, break line

File: practice-01/styles/font.css [hunk 4]
```diff
@@ -16,13 +16,26 @@
...
+  color: var(--west_chester-grey);
+}
+.text--white{
```
Comment (phattruong-agilityio) [inline practice-01/styles/font.css]: code style, break line

File: practice-01/styles/font.css [hunk 5]
```diff
@@ -16,13 +16,26 @@
...
+  color: var(--white);
+}
+.text--black, .text--black .item{
```
Comment (phattruong-agilityio) [inline practice-01/styles/font.css]: why have `.item` here

File: practice-01/styles/index.css [hunk 6]
```diff
@@ -31,12 +31,48 @@ a {
...
+  cursor: pointer;
 
   font-weight: var(--font-weight-bold);
   font-size: var(--font-size-medium);
   line-height: var(--line-height-small);
 }
+.btn--primary {
+  color: var(--black);
+}
+.btn--primary:hover {
```
Comment (phattruong-agilityio) [inline practice-01/styles/index.css:46]: code style, break line

File: practice-01/styles/index.css [hunk 7]
```diff
@@ -31,12 +31,48 @@ a {
...
+.btn--primary:active {
+  background-color: var(--crab-nebula);
+  transform: translateY(2px);
+  color: var(--white);
+}
 
+.btn--secondary:hover {
+  opacity: 0.8;
+}
+.btn--secondary:active {
```
Comment (phattruong-agilityio) [inline practice-01/styles/index.css:61]: code style, break line. Please CHECK ALL

**Review by phattruong-agilityio — APPROVED** [review]

File: practice-01/styles/variable.css [hunk 8]
```diff
@@ -24,10 +24,10 @@
 
   /* -------------font-size---------------- */
   --font-size-small: 14px; /*h6*/
-  --font-size-datetime: 16px; /*h5*/
+  --font-size-date_time: 16px; /*h5*/
```
Comment (phattruong-agilityio) [inline practice-01/styles/variable.css]: datetime is wrong. Please using size metric naming

**Pull request merged**